lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -125,6 +125,13 @@ ParseSILModuleRequest::evaluate(Evaluator &evaluator,
            "Failed to parse SIL but did not emit any errors!");
     return SILModule::createEmptyModule(desc.context, desc.conv, desc.opts);
   }
+
+  for (auto Fn : parserState.PotentialZombieFns) {
```

Comment:
The destructor is getting called too late. I will refactor this into a separate function